### ISSUE TITLE
Builds are failing on AppVeyor on IncludeInterceptorTest 

### ISSUE THREAD

**Issue by sanmai** [infection/infection#395]
| Question    | Answer
| ------------| ---------------
| Infection version | master
| Platform    | Windows, AppVeyor

```
1) Infection\Tests\StreamWrapper\IncludeInterceptorTest::test_it_throws_an_exception_if_not_configured
PHPUnit\Framework\Exception: Failed loading C:\tools\php\ext\xdebug.dll
```

Surprisingly, builds on PHP 7.0 are not affected.

**Comment by BackEndTea** [comment]
It looks like it has the php 7 version of xdebug cached, and threfore doesn't download the newer versions.

Why it fails on that specific test, i dont' know.

**Comment by BackEndTea** [comment]
This has been fixed by #400 & #410
